app/api/v1/web_socket.py

```
@@ -58,6 +66,7 @@ async def websocket_endpoint(
     llm_service: LLMService = Depends(get_llm_service),
     stt_service: STTService = Depends(get_stt_service),
     tts_service: TTSService = Depends(get_tts_service),
+    schedule_service: ScheduleService = Depends(get_schedule_service),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)